huaweicloud/services/dcs/resource_huaweicloud_dcs_instance.go

```diff
@@ -836,9 +958,61 @@ func resourceDcsInstancesRead(_ context.Context, d *schema.ResourceData, meta in
 		d.Set("whitelist_enable", wList.Enable),
 	)
 
+	if diagErr := setRdsInstanceParameters(ctx, d, client, d.Id()); diagErr != nil {
+		return diagErr
+	}
```

Comment:
this will hide the errors from multierror please consider this:
```
diagErr = append(diagErr, diag.FromErr(mErr.ErrorOrNil())...)
return diagErr
```

Reply:
done